compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -321,7 +321,7 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         withDefaultPos(tpd.TypeDef(symbol.asType))
       def copy(original: Tree)(name: String, rhs: Tree): TypeDef =
         tpd.cpy.TypeDef(original)(name.toTypeName, rhs)
-      def unapply(tdef: TypeDef): Option[(String, Tree /*TypeTree | TypeBoundsTree*/ /* TypeTree | TypeBoundsTree */)] =
+      def unapply(tdef: TypeDef): Some[(String, Tree /*TypeTree | TypeBoundsTree*/ /* TypeTree | TypeBoundsTree */)] =
```

Comment:
can't we just use tuples and not wrap with `Some`?

Reply:
That is what we said in the comment above

Reply:
oh yes :)